collectors/cmdb/hpsm/pom.xml

```diff
@@ -140,6 +140,11 @@
       <artifactId>spring-boot-starter-test</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.json</groupId>
+      <artifactId>json</artifactId>
+      <version>20180130</version>
+    </dependency>
```

Comment:
where are you using org.json?

Reply:
I am using this in DefaultHpsmClient.java. It allows me to easily convert the XML soap response to JSON.

Reply:
Any other options? I dont like org.json license!

Reply:
Okay, I will remove it and create my own class for the conversion.